src/viewer.js

```diff
@@ -3990,8 +4006,57 @@ function doViewerResize(viewer, containerSize){
     THIS[viewer.hash].needsResize = false;
     THIS[viewer.hash].forceResize = false;
 }
+
+function handleArrowKeys(viewer) {
+    // Use the viewer's configured pan amount
+    const pixels = viewer.pixelsPerArrowPress || 40;
```

Comment:
Having the fallback here is nice defensive coding, but it would also mask the problem if for some reason that was missing; I think it would be better to leave it off so that we know about the problem if it happens.